eng/code-quality-reports/src/test/resources/com/azure/tools/checkstyle/checks/GoodLoggerCheck/InvalidLoggerNameTestData.java

```diff
@@ -0,0 +1,8 @@
+import com.azure.core.util.logging.ClientLogger;
+
+public class InvalidLoggerNameTestData {
+    // invalid LOGGER name
+    private final ClientLogger LOGGER = new ClientLogger(InvalidLoggerNameTestData.class);
```

Comment:
Do we need to check this one as the built-in Checkstyle rule will throw an error here about incorrect casing

Reply:
Thanks. Will change the var name to others, like "abcdLogger"